src/site/content/_layout.njk

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
I feel like we should give this file a better name so it's not confused with the layout in `_includes`.

Bikeshed ideas... "sw-partial-layout.njk" ?